### PR TITLE
Miscellaneous fixes

### PR DESCRIPTION
**Pull request by detiber**
## Description

Various fixes

## Why is this needed

- Fixes metadata.yaml to match current version information in repo
- Adds clusterctl-settings.json to facilitate easier testing with clusterctl
- Adds tilt-provider.json to facilitate easier testing with Tilt
- Fixes the image name in config/manager/manager_image_patch.yaml to remove an extraneous `.io`
- Updates the name prefix used when generating manifests from cluster-api-provider-tinkerbell- to capt- to avoid issues with name length
- Regenerated manifests using `make crds` to pick up the above changes and also reflect the current state of things

Fixes: #11

### CODE REVIEW

File: clusterctl-settings.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+    "name": "infrastructure-tinkerbell",
```
Comment (invidian) [inline clusterctl-settings.json:2]: Do we want it `infrastructure-tinkerbell` or `infrastructure-tink` ? #11

File: clusterctl-settings.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+    "config": {
+      "componentsFile": "infrastructure-components.yaml",
+      "nextVersion": "v0.0.0"
+    }
+}
```
Comment (invidian) [inline clusterctl-settings.json:7]: Missing newline?

File: tilt-provider.json [hunk 3]
```diff
@@ -0,0 +1,9 @@
+{
+    "name": "tinkerbell",
+    "config": {
+        "image": "quay.io/tinkerbell/cluster-api-provider-tinkerbell",
+        "live_reload_deps": [
+            "main.go", "go.mod", "go.sum", "api", "controllers"
+        ]
+    }
+}
```
Comment (invidian) [inline tilt-provider.json:9]: Ditto missing newline?

File: config/manager/manager_image_patch.yaml [hunk 4]
```diff
@@ -8,5 +8,5 @@ spec:
     spec:
       containers:
       # Change the value of image field below to your controller image URL
-      - image: quay.io.io/tinkerbell/cluster-api-provider-tinkerbell:e2e
```
Comment (invidian) [inline config/manager/manager_image_patch.yaml:11]: Hehe, I run into this one as well :smile:

File: clusterctl-settings.json (hunk 1 above)
Comment (invidian) [reply clusterctl-settings.json:2]: Oh, I see next commits now.

File: config/managerless/kustomization.yaml [hunk 5]
```diff
@@ -1,11 +1,11 @@
 # Adds namespace to all resources.
-namespace: cluster-api-provider-tinkerbell-system
+namespace: capt-system
```
Comment (invidian) [inline config/managerless/kustomization.yaml:2]: Perhaps that fixes #11 ?

**Review by invidian — CHANGES_REQUESTED** [review]
LGTM, just some nits and questions.

File: config/managerless/kustomization.yaml (hunk 5 above)
Comment (detiber) [reply config/managerless/kustomization.yaml:2]: Yes, this was a fix for #11

File: clusterctl-settings.json (hunk 1 above)
Comment (detiber) [reply clusterctl-settings.json:2]: I think we want tinkerbell here, since this will be used by clusterctl for output.

**Comment by detiber** [comment]
@invidian this should be good to go now

**Review by invidian — APPROVED** [review]
LGTM

**Pull request merged**